### PR TITLE
Add recipe for aiozmq 0.7.1.

### PR DESCRIPTION
**Pull request by acaprez**

### CODE REVIEW

File: recipes/aiozmq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - msgpack-python >=0.4.0
+
+test:
+  imports:
+    - aiozmq
+    - aiozmq.cli
+    - aiozmq.rpc
+
+  commands:
+    - aiozmq-proxy --help  # [not win]
```
Comment (mariusvniekerk) [inline recipes/aiozmq/meta.yaml]: You might want to add this as an entrypoint

Comment (acaprez) [reply recipes/aiozmq/meta.yaml]: It's already specified as an entrypoint in the package's [setup.py](https://github.com/aio-libs-abandoned/aiozmq/blob/v0.7.1/setup.py#L72).

File: recipes/aiozmq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<33]
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (jakirkham) [inline recipes/aiozmq/meta.yaml]: Please add...

```yaml
  entry_points:
    - aiozmq-proxy = aiozmq.cli.proxy:main
```

cc @mariusvniekerk

File: recipes/aiozmq/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/aiozmq/meta.yaml]: Made this *hopefully* more clear [above]( https://github.com/conda-forge/staged-recipes/pull/5561/files#r180284721 ).

**Pull request closed without merging**